code/modules/religion/rites.dm

```diff
@@ -57,17 +68,24 @@
 	invoke_msg = "... Arise, our champion! Become that which your soul craves, live in the world as your true form!!"
 	favor_cost = 500
 
-/datum/religion_rites/synthconversion/perform_rite(mob/living/user, obj/structure/altar_of_gods/AOG)
-	if(!AOG?.buckled_mobs?.len)
-		to_chat(user, "<span class='warning'>This rite requires an individual to be buckled to [AOG].</span>")
+/datum/religion_rites/synthconversion/perform_rite(mob/living/user, atom/religious_tool)
+	if(!ismovable(religious_tool))
+		to_chat(user, "<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")
+		return FALSE
+	var/atom/movable/religious_tool2
+	if(!religious_tool2?.buckled_mobs?.len)
+		to_chat(user, "<span class='warning'>This rite requires an individual to be buckled to [religious_tool].</span>")
 		return FALSE
 	return ..()
 
-/datum/religion_rites/synthconversion/invoke_effect(mob/living/user, obj/structure/altar_of_gods/AOG)
-	if(!AOG?.buckled_mobs?.len)
+/datum/religion_rites/synthconversion/invoke_effect(mob/living/user, atom/religious_tool)
+	if(!ismovable(religious_tool))
```

Comment:
wouldn't perform rite already confirm this? or is it just for sanity's sake

Reply:
you're right but i will keep this in for futureproofing when someone messes up my BEAUTIFUL code

Reply:
Make it `CRASH()` and scream then?

Reply:
i only accept indoor voices, sorry